### PR TITLE
Add benchmark for global bucket

### PR DESCRIPTION
**Pull request by Sbozzolo**
This PR adds flamegraphs for allocations and compute for the global temporal bucket runs.

### CODE REVIEW

File: experiments/standalone/Bucket/global_bucket_temporalmap.jl [hunk 1]
```diff
@@ -38,6 +38,14 @@ using ClimaLand:
     PrescribedRadiativeFluxes
 using ClimaLand.TimeVaryingInputs
 
+PROFILING = false
+try
```
Comment (juliasloan25) [inline experiments/standalone/Bucket/global_bucket_temporalmap.jl:42]: why is this in a try catch? so we only profile on buildkite when we've added ProfileCanvas to the environment?

**Review by juliasloan25 — APPROVED** [review]
This looks good to me! It'll be nice to have it for the most complex albedo case

File: experiments/standalone/Bucket/global_bucket_temporalmap.jl (hunk 1 above)
Comment (Sbozzolo) [reply experiments/standalone/Bucket/global_bucket_temporalmap.jl:42]: I don't want to add ProfileCanvas to experiments environment, and users should be able to run the code outside of buildkite

**Review by juliasloan25 — COMMENTED** [review]
I almost forgot! Please also update the NEWS.md to explain these changes :) I'll send a message to the land channel

**Pull request merged**